src/Layout/redist/redist.csproj

```diff
@@ -47,7 +47,6 @@
 
     <!-- Lift up dependencies of dependencies to prevent build tasks from getting pinned to older versions -->
     <PackageReference Include="System.CodeDom" />
-    <PackageReference Include="System.Text.Encoding.CodePages" />
```

Comment:
do we no longer need to lift up this dependency?

Reply:
Not necessary anymore as transitive package dependencies that are already provided by the framework with a higher or equal version get pruned during restore. The inbox assembly gets used instead.